src/WebJobs.Script/Description/DocumentDBBindingMetadata.cs

```diff
@@ -25,9 +22,9 @@ public override void ApplyToConfig(JobHostConfigurationBuilder configBuilder)
             }
 
             DocumentDBConfiguration config = new DocumentDBConfiguration();
-            if (!string.IsNullOrEmpty(ConnectionString))
+            if (!string.IsNullOrEmpty(Connection))
             {
-                config.ConnectionString = ConnectionString;
+                config.ConnectionString = Utility.GetAppSettingOrEnvironmentValue(Connection);
             }
 
```

Comment:
So what does it mean to call UseDocumentDB(config) multiple times with different configs? I know this will be changing soon, but curious as to how this is handled.